src/Hipace.cpp

```diff
@@ -121,9 +121,17 @@ Hipace::Hipace () :
     if (amrex::TilingIfNotGPU()) {
         DfltMfiTlng.EnableTiling();
     }
-    queryWithParser(pph, "external_ExmBy_slope", m_external_ExmBy_slope);
-    queryWithParser(pph, "external_Ez_slope", m_external_Ez_slope);
-    queryWithParser(pph, "external_Ez_uniform", m_external_Ez_uniform);
+
```

Comment:
```suggestion
DeprecatedInput("hipace", "external_ExmBy_slope", "external_E_slope");
DeprecatedInput("hipace", "external_Ez_slope", "external_E_slope");
DeprecatedInput("hipace", "external_Ez_uniform", "external_E_uniform");
```

Although they were never documented, should we deprecate the previous ones?